spec/ruby/core/process/status/bit_and_spec.rb

```diff
@@ -1,35 +1,38 @@
 require_relative '../../../spec_helper'
 
-describe "Process::Status#&" do
-  it "returns a bitwise and of the integer status of an exited child" do
-    suppress_warning do
-      ruby_exe("exit(29)", exit_status: 29)
-      ($? & 0).should == 0
-      ($? & $?.to_i).should == $?.to_i
+ruby_version_is ""..."3.5" do
```

Comment:
```suggestion
ruby_version_is "3.3"..."3.5" do
```

And remove `ruby_version_is "3.3" do` bellow.